sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/PhpUnitSensor.java

```diff
@@ -110,20 +108,9 @@ private File getIOFile(String path) {
     return file;
   }
 
-  /**
-   * {@inheritDoc}
-   */
-  @Override
-  public boolean shouldExecuteOnProject(Project project) {
-    return fileSystem.hasFiles(filePredicates.and(filePredicates.hasLanguage(Php.KEY), filePredicates.hasType(InputFile.Type.MAIN)));
-
-  }
-
-  /**
-   * {@inheritDoc}
-   */
   @Override
   public String toString() {
```

Comment:
@vilchik-elena I renamed PHPUnitSensor into PhpUnitService, as it is no longer a sensor. The new class PhpUnitService has no method toString(). Maybe we are not talking about the same thing.